RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -24,306 +24,33 @@ import SwiftUI
-    private(set) var purchasesProvider: CustomerCenterPurchasesType
+    private(set) var purchasesActive: [PurchaseInformation] = []
 
     init(
         screen: CustomerCenterConfigData.Screen,
         actionWrapper: CustomerCenterActionWrapper,
-        purchaseInformation: PurchaseInformation?,
+        purchasesActive: [PurchaseInformation] = [],
         refundRequestStatus: RefundRequestStatus? = nil,
         purchasesProvider: CustomerCenterPurchasesType,
         loadPromotionalOfferUseCase: LoadPromotionalOfferUseCaseType? = nil) {
-            self.screen = screen
-            self.paths = screen.filteredPaths
-            self.purchaseInformation = purchaseInformation
-            self.purchasesProvider = purchasesProvider
-            self.refundRequestStatus = refundRequestStatus
-            self.actionWrapper = actionWrapper
-            self.loadPromotionalOfferUseCase = loadPromotionalOfferUseCase
-            ?? LoadPromotionalOfferUseCase(purchasesProvider: purchasesProvider)
-            self.state = .success
-            self.restoreAlertType = .loading
-        }
-
-    func reloadPurchaseInformation(_ purchaseInformation: PurchaseInformation) {
-        self.purchaseInformation = purchaseInformation
-    }
-
-#if os(iOS) || targetEnvironment(macCatalyst)
-    func determineFlow(for path: CustomerCenterConfigData.HelpPath, activeProductId: String? = nil) async {
-        // Convert the path to an appropriate action using the extension
-        if let action = path.asAction() {
-            // Send the action through the action wrapper
-            self.actionWrapper.handleAction(.buttonTapped(action: action))
-        }
-
-        switch path.detail {
-        case let .feedbackSurvey(feedbackSurvey):
-            self.feedbackSurveyData = FeedbackSurveyData(
-                configuration: feedbackSurvey,
-                path: path) { [weak self] in
-                    Task {
-                        await self?.onPathSelected(path: path)
-                    }
-                }
-
-        case let .promotionalOffer(promotionalOffer):
-            if promotionalOffer.eligible {
-                self.loadingPath = path
-                let result = await loadPromotionalOfferUseCase.execute(promoOfferDetails: promotionalOffer)
-                switch result {
-                case .success(let promotionalOfferData):
-                    self.promotionalOfferData = promotionalOfferData
-                case .failure:
-                    await self.onPathSelected(path: path)
-                    self.loadingPath = nil
-                }
-            } else {
-                Logger.debug(Strings.promo_offer_not_eligible_for_product(
-                    promotionalOffer.iosOfferId, activeProductId ?? ""
-                ))
-                await self.onPathSelected(path: path)
-            }
-
-        default:
-            await self.onPathSelected(path: path)
-        }
-    }
-
-    func handleSheetDismiss() async {
```

Comment:
where's this?

Reply:
I think this wasn't used before? Because I can't find where it was being called

Reply:
Exactly, this was not being used so I figured it was safe to remove it